rasa/cli/interactive.py

```diff
@@ -107,8 +107,8 @@ def check_training_data(args):
         get_validated_path(f, "data", DEFAULT_DATA_PATH, none_is_valid=True)
         for f in args.data
     ]
-    story_directory, nlu_data_directory = data.get_core_nlu_directories(training_files)
-    if not os.listdir(story_directory) or not os.listdir(nlu_data_directory):
+    story_files, nlu_files = data.get_core_nlu_files(training_files)
```

Comment:
Why was this changed?

Reply:
we don't need to move this in a tmp directory, we can just check whether any files were found.